src/themes/function.js

```diff
@@ -1,17 +1,32 @@
 import { css } from 'styled-components'
 
-export const Margins = css`
-    margin: ${(props) => (props.m ? props.m : '')};
-    margin-top: ${(props) => (props.mt ? props.mt : '')};
-    margin-bottom: ${(props) => (props.mb ? props.mb : '')};
-    margin-right: ${(props) => (props.mr ? props.mr : '')};
-    margin-left: ${(props) => (props.ml ? props.ml : '')};
+export const Margins = ({ m, mt, ml, mr, mb }) => css`
```

Comment:
Margins and Paddings function are used in many (Box, Text, Button) components, can you make a different function for it instead of overwrite the current one

Reply:
Actually, it does not affect the code. I will explain in detail.